src/session/session_api.c

```diff
@@ -444,18 +444,11 @@ err:		if (cursor != NULL)
 	}
 
 	/*
-	 * Opening a cursor on a non-existent data source will set ret to
```

Comment:
This block of comment should still be present as it explains that it is possible to get either ENOENT or WT_NOTFOUND when opening a cursor. And hence it would be required to map the WT_NOTFOUND to ENOENT in CURSOR_OPEN_API_END.